packages/web/src/components/table/Table.tsx

```diff
@@ -152,6 +159,9 @@ export const Table = ({
   // Pagination page
   const [currentPage, setCurrentPage] = useState<number>(0)
   const maxPage = useMemo(() => {
+    if (totalRowCount == null) {
+      return 0
```

Comment:
since `totalRowCount` doesn't have a default value anymore, this prevents this value from breaking